packages/expo-image/android/src/main/java/com/caverock/androidsvg/SVGStyler.kt

```diff
@@ -0,0 +1,43 @@
+package com.caverock.androidsvg
+
+internal fun replaceColour(paint: SVG.SvgPaint?, newColour: Int) {
```

Comment:
To be honest I would rather use american english in our own codebase, which means `colour` -> `color`